src/test/java/uk/nhs/hee/tis/trainee/forms/migration/ConvertObjectIdsToUuidStringsTest.java

```diff
@@ -89,6 +93,18 @@ void setUp() {
 
     when(template.getCollectionName(FormRPartA.class)).thenReturn(PART_A_COLLECTION_NAME);
     when(template.getCollectionName(FormRPartB.class)).thenReturn(PART_B_COLLECTION_NAME);
+
+    MappingMongoConverter converter = mock(MappingMongoConverter.class);
+    when(converter.read(any(), any())).thenAnswer(args -> {
+      Class<? extends AbstractForm> formClass = args.getArgument(0);
+      Document document = args.getArgument(1);
+
+      AbstractForm form = formClass.getConstructor().newInstance();
+      form.setId(document.get(ID_FIELD).toString());
+      form.setTraineeTisId(document.getString(TRAINEE_ID_FIELD));
```

Comment:
Is it just the two lines above that will need to be rewritten later (I assume we're not changing the DTO id to UUID as well)?

Reply:
Yeah, just line 103.

I'm a bit undecided about the DTO, we can change it without any issues and it doesn't technically break the contract since it gets converted to a string in the JSON response.

If we change the DTO then the API call will fail instantly if the provided ID isn't UUID compatible, if we don't change the DTO then it would fail during the conversion from DTO to Entity instead... so it really comes down to where we want validation/failure to occur.